app/connectors/UpscanInitiateConnector.scala

```diff
@@ -35,6 +36,8 @@ import javax.inject.{Inject, Singleton}
 class UpscanInitiateConnector @Inject() (appConfig: FrontendAppConfig, http: HttpGet with HttpPost, metrics: Metrics)
     extends HttpAPIMonitor {
 
+  private val logger = LoggerFactory.getLogger("application." + getClass.getCanonicalName)
```

Comment:
Used the same style as existing "loggers" in this project.
But could replace all with `Logger(this.getClass)` if we want to